akka-actor-testkit-typed/src/main/scala/akka/actor/testkit/typed/CapturedLogEvent.scala

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (C) 2018 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.actor.testkit.typed
+
+import java.util.Optional
+
+import akka.actor.typed.LogMarker
+import akka.event.Logging.LogLevel
+import akka.util.OptionVal
+import akka.util.OptionVal.{ None, Some }
+
+import scala.collection.JavaConverters._
+import scala.compat.java8.OptionConverters._
+import scala.{ None ⇒ SNone, Option ⇒ SOption, Some ⇒ SSome }
+
+/**
+ * Representation of a Log Event issued by a [[akka.actor.typed.Behavior]]
+ */
+final case class CapturedLogEvent private[akka] (logLevel: LogLevel, message: String,
+                                                 private val cause:  OptionVal[Throwable],
+                                                 private val marker: OptionVal[LogMarker],
```

Comment:
I don't think we need to use OptionVal in testkit, it's intended as a performance optimization

Reply:
OK, then I'll use Scala's `Option` in here and do the `OptionVal` to `Option` conversion in here: https://github.com/akka/akka/blob/master/akka-actor-testkit-typed/src/main/scala/akka/actor/testkit/typed/internal/StubbedActorContext.scala#L76